lib/screens/home/home_view_model.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:async';
+
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:intl/intl.dart';
+import 'package:survey_flutter/model/survey_model.dart';
+import 'package:survey_flutter/model/profile_model.dart';
+import 'package:survey_flutter/usecases/base/base_use_case.dart';
+
+final homeViewModelProvider =
+    AsyncNotifierProvider.autoDispose<HomeViewModel, dynamic>(
+        HomeViewModel.new);
+
+class HomeViewModel extends AutoDisposeAsyncNotifier {
```

Comment:
```suggestion
class HomeViewModel extends AutoDisposeAsyncNotifier<void> {
```
If there is no specific data type, please use `void`